src/app/page.tsx

```diff
@@ -1,13 +1,53 @@
-import { HydrateClient } from '~/trpc/server';
+import { Button } from "~/components/ui/Button";
+import { HydrateClient } from "~/trpc/server";
+
+import Image from "next/image";
+import TestIcon from "/public/icons/ico.png";
 
 export default async function Home() {
-	return (
-		<HydrateClient>
-			<main>
-				<h1 className="font-extrabold text-3xl text-primary-100">Attios CRM</h1>
-				<h2 className="font-normal text-black text-xs">Attios CRM</h2>
-				<h3 className="font-light text-base text-primary-200">Attios CRM</h3>
-			</main>
-		</HydrateClient>
-	);
+  return (
+    <HydrateClient>
+      <main className="flex max-w-48 flex-col gap-1">
+        <h1 className="font-extrabold text-3xl text-primary-100">Attios CRM</h1>
+        <Button>
+          Click me
+          <Image src={TestIcon} alt="Test Icon" />
+        </Button>
+        <Button>
+          <Image src={TestIcon} alt="Test Icon" />
+        </Button>
+        <Button color="secondary">Click me</Button>
+        <Button color="tertiary">Click me</Button>
```

Comment:
Esse botão tá disabled mesmo que a gente não esteja colocando ele para ser disabled.
No color tertiary tá com um hover:cursor-not-allowed.

Isso deveria ser disabled:cursor-not-allowed. E deveria estar para todas as versões do botão.
Além do que, botões disabled deveriam ter um estilo diferente, um bg diferente